tchannel-crossdock/docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: '2'
+
+services:
+    crossdock:
+        image: crossdock/crossdock
+        links:
+            - go
+            - java
+            - python
+        environment:
+            - WAIT_FOR=go,java,python
+
+            - AXIS_CLIENT=go
```

Comment:
I should add a readme, probably. This is a _crossdock client_, the orchestrator of the behaviors, and it's only implemented in Go. Each languages implements only the "server" part.